go.mod

```diff
@@ -1,159 +1,116 @@
 module github.com/DataDog/watermarkpodautoscaler
 
-go 1.21
+go 1.22.0
+
+toolchain go1.22.7
 
 require (
-	github.com/DataDog/datadog-operator v1.3.0
-	github.com/go-logr/logr v1.3.0
+	github.com/DataDog/datadog-operator v0.7.1-0.20241003091406-6e9e14d97438
```

Comment:
does this tag correspond to main?
can we use `v1.8.0` ?

Reply:
This tag corresponds to the opened PR DataDog/datadog-operator#1445 because I preferred to have all the dependencies aligned and avoid having a bunch of different versions for a given dependency (like `k8s.io` modules) depending on the path that pulls it.

So, this will need to be updated to a `main` commit as soon as DataDog/datadog-operator#1445 will be merged.